tests/unit/test_routes.py

```diff
@@ -246,6 +246,33 @@ def test_categories(module_client, module_db):
     assert (response.status_code == 404)
     assert (response.json.get('errors')[0].get('code') == "not-found")
 
+def test_get_single_category(module_client, module_db):
+    client = module_client
+
+    response = client.get('api/v1/categories/4')
+
+    # Status should be OK
+    assert (response.status_code == 200)
+
+    category = response.json['data']
+    assert (isinstance(category.get('name'), str))
+    assert (category.get('name') == 'Tutorials')
```

Comment:
We don't need this assertion, it too tightly couples the data with the test. If we were to change the yaml file, for example, `Tutorials` might not end up as the fourth category. Just checking that it's a string with ID 4 is fine